Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -454,7 +453,7 @@ fileprivate extension _EasyHandle {
             }
             var length = Double()
             try! CFURLSession_easy_getinfo_double(handle.rawHandle, CFURLSessionInfoCONTENT_LENGTH_DOWNLOAD, &length).asError()
```

Comment:
Just an aside... This API returns the ContentLength as a `Double`, which seems crazy. There is a [newer API](https://curl.haxx.se/libcurl/c/CURLINFO_CONTENT_LENGTH_DOWNLOAD_T.html) which has a better return type. It was introduced in libcurl 7.55.0; could we switch to it instead?